app/views/details.scala.html

```diff
@@ -245,6 +245,41 @@ <h1>
             </small>
         </h1>
     </div>
+    @if(resource.getType.contains("DifferentiatedPerson") && resource.getBirthYear > 1940 && resource.getDeathYear == null){
+    <div id="meta-person" class="panel panel-info">
```

Comment:
I think the `panel-info` color doesn't look right with our colors. To use our brand color, we can use `panel-primary`, but that is too dark and prominent in my opinion. I think we should use `panel-default`:

```suggestion
    <div id="meta-person" class="panel panel-default">
```